package.json

```diff
@@ -26,7 +26,7 @@
     "update-readme": "node scripts/buildtools update-readme",
     "retirejs": "retire --jspath target/js --package",
     "lint:eslint": "eslint .",
-    "lint:eslint:report": "eslint -f checkstyle -o build2/OSW-eslint-checkstyle-result.xml .",
+    "lint:eslint:report": "eslint -f checkstyle -o build2/OSW-eslint-checkstyle-result.xml . --quiet",
```

Comment:
Required now that we're reporting warnings for inclusive language (`.*master.*`) due to [this change](https://github.com/okta/okta-signin-widget/blob/master/packages/%40okta/i18n/src/properties/login.properties#L210).